packet/flatcar-linux/kubernetes/variables.tf

```diff
@@ -21,12 +21,24 @@ variable "project_id" {
 
 # Nodes
 
+variable "os_arch" {
+  type        = "string"
+  default     = "amd64"
+  description = "Flatcar Container Linux architecture to install (amd64, arm64)"
+}
+
 variable "os_channel" {
   type        = "string"
   default     = "stable"
   description = "Flatcar Container Linux channel to install from (stable, beta, alpha, edge)"
 }
 
+variable "os_version" {
+  type        = "string"
+  default     = "current"
+  description = "Flatcar Container Linux version to install (for example '2191.5.0' - see https://www.flatcar-linux.org/releases/)"
```

Comment:
Here it should be also mentioned, that this is only for iPXE installation.

Reply:
done